packages/react/src/form-field/FormField.tsx

```diff
@@ -0,0 +1,74 @@
+import * as RadixLabel from "@radix-ui/react-label";
+import { useId } from "@reach/auto-id";
+import clsx from "clsx";
+import {
+  type ComponentPropsWithRef,
+  type ReactElement,
+  cloneElement,
+  forwardRef,
+} from "react";
+
+import type { ExtendProps } from "../utils/ExtendProps";
+
+import { Box } from "../box";
+import { Text } from "../text";
+import * as styles from "./FormField.css";
+type FormGroupProps = ExtendProps<
+  ComponentPropsWithRef<typeof Box>,
+  {
+    children: ReactElement;
+    description?: string;
+    disabled?: boolean;
+    error?: string;
+    id?: string;
+    label?: string;
+    required?: boolean;
+  }
+>;
+
+export const FormField = forwardRef<HTMLDivElement, FormGroupProps>(
+  (
+    {
+      children,
+      className,
+      description,
+      disabled,
+      error,
+      id: _id,
```

Comment:
let's name this `idProp`
```suggestion
      id: idProp,
```

so it's clear that this is a prop value that we've renamed for a particular reason